migration/migrator/migrations/system/20250710102303_php_ppa.py

```diff
@@ -0,0 +1,47 @@
+"""Migration for the Submitty system."""
+
+import os
+
+
+modules = [
+    "cli",
+    "fpm",
+    "curl",
+    "pgsql",
+    "zip",
+    "mbstring",
+    "xml",
+    "ds",
+    "imagick",
+    "intl",
+]
+
+
+def up(config):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    """
+    os.system("apt-get remove -qqy php*")
```

Comment:
A sysadmin using LdapAuthentication who has `php-ldap` installed will now have a broken authentication after running this migration. We should try to write this a little more gracefully and consider any possible PHP extensions. xdebug will also be one for the dev side which should stick around too